packages/manager/apps/dedicated/client/app/dedicated/server/install/image/components/config/controller.js

```diff
@@ -0,0 +1,37 @@
+import head from 'lodash/head';
+
+export default class DedicatedServerInstallImageConfigCtrl {
+  addHttpHeader() {
+    this.model.httpHeader.push({
+      added: false,
+      model: {},
+    });
+  }
+
+  /*= =============================
+  =            Events            =
+  ============================== */
+
+  onHttpHeaderAddBtnClick(index) {
+    this.model.httpHeader[index].added = true;
+    this.addHttpHeader();
+  }
+
+  onHttpHeaderDeleteBtnClick(index) {
+    this.model.httpHeader.splice(index, 1);
+  }
+
+  /* -----  End of Events  ------*/
+
+  /*= =====================================
+  =            Initialization            =
+  ====================================== */
+
+  $onInit() {
```

Comment:
Maybe `$onInit` should be moved as the first method here ? Imo it is easier to read and maintain when we have first constructor (to know what is injected) and then what is done at the initialization